awscli/customizations/cloudformation/artifact_exporter.py

```diff
@@ -432,6 +445,21 @@ def __init__(self, template_path, parent_dir, uploader,
         self.resources_to_export = resources_to_export
         self.uploader = uploader
 
+    def export_global_artifacts(self, template_dict):
+        """
+        Template params such as AWS::Include transforms are not specific to 
+        any resource type but contain artifacts that should be exported,
+        here we iterate through the template dict and export params with a 
+        handler defined in GLOBAL_EXPORT_DICT
+        """
+        for key, val in iter(template_dict.items()):
+            if key in GLOBAL_EXPORT_DICT:
+                template_dict[key] = GLOBAL_EXPORT_DICT[key](val, self.uploader)
+            elif type(val) is dict:
+                self.export_global_artifacts(val)
+        return self.template_dict
```

Comment:
Is it intentional that we are returning `self.template_dict`? I would assume it would instead be `return template_dict`. We probably we do not even need to return anything here because Python dictionaries are mutable so any updates made to the dictionary will reflect in the original dictionary passed in.